src/__tests__/builder.spec.ts

```
@@ -108,7 +108,8 @@ describe('SearchQueryToOpenSearchFilterTranslator', () => {
 			.query as OpenSearchVectorQuery;
 
 		expect(query.knn.embeddingField.vector).toEqual(embedding);
-		expect(query.knn.embeddingField.filter.bool.must).toEqual([
+		expect(query.knn.embeddingField.filter).toBeDefined();
+		expect(query.knn.embeddingField.filter!.bool.must).toEqual([
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)